### DISCUSSION TITLE
Render Cron job

### DISCUSSION THREAD

**Discussion by m0rdreck** [blitz-js/blitz#3322] [Q&A]
Hello

I try to use Render cron job to execute one big mutation the blitz console is open, but my mutation is not executing.

Can you help me plz ?

Thank you

```
const {streamWrite, streamEnd, onExit} = require('@rauschma/stringio');
import { spawn } from 'child_process'

async function updateAgentsGroupes() {
  const blitzProcess = spawn('blitz console', [], { stdio: ['pipe', process.stdout, process.stderr] })
  blitzProcess.on('error', (err) => {
    console.error(err.message)
  })

  writeToWritable(blitzProcess.stdin);

  await onExit(blitzProcess);
}

async function writeToWritable(writable) {
  await streamWrite(writable, 'await updateGroup()\n')
  await streamEnd(writable);
}

updateAgentsGroupes()
```

**Comment by beerose** [comment]
Hmm, I'm not sure how Render's cron jobs work exactly, but in the code you sent it looks like `updateGroup` is not imported anywhere. Also, you might want to use `invokeWithMiddleware` (imported from `blitz`) to execute the mutation: https://blitzjs.com/docs/resolver-server-utilities#invoke-with-middleware
